prereise/gather/demanddata/bldg_electrification/floor_area_avg_slopes.py

```diff
@@ -0,0 +1,608 @@
+import os
+
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from zone_profile_generator import zone_shp_overlay
+
+from prereise.gather.demanddata.bldg_electrification import const
+
+
+def get_zone_floor_area(iso):
+    """Computes the zone floor area for each ISO.
+
+    :param str iso: abbrev. name of ISO.
+    :return: (*pandas.DataFrame*) -- Floor area in square meters for all the zones with breakdowns of residential and commercial, total heating and total cooling
+
+    ..note:: zone floor area in square meters saved as csv into Profiles/result_stats
```

Comment:
We will need a space after "..", i.e. ".. note:: ", to make `Sphinx` render correctly. Also, an indent is needed for notes longer than one line, i.e. 
```python
.. note:: This is a long note. This is a long note. This is a long note.
    This is a long note. End.
```

Reply:
Made the changes

Reply:
Made the changes